.github/workflows/release_and_publish.yml

```diff
@@ -5,6 +5,7 @@ on:
     # Sequence of patterns matched against refs/tags
     tags:
       - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+  workflow_dispatch:  # Enables manual triggering from GH UI
```

Comment:
I think we need the remaining arguments like over [in `libpysal`](https://github.com/pysal/libpysal/blob/9479b34f9acbfcc0af9c20c68046aaa8019e38aa/.github/workflows/unittests.yml#L12C1-L16C24):

```
  workflow_dispatch:
    inputs:
      version:
        description: Manual Unittest Run
        default: test
        required: false
```